phpBB/styles/prosilver/theme/common.css

```diff
@@ -603,6 +604,53 @@ a.header-avatar span:after {
 	display: none !important;
 }
 
+/* Smiley Dropdown---------------------------------------- */
+.smiley-box {
+	float: right;
+	width: 100%;
+}
+
+.smiley-box .smiley-image {
+	margin: 3px;
+}
+
+.dropdown .smiley-content {
+	width: 200px;
+}
+
+.smiley-box .smiley-list-item {
+	border: none;
+	display: inline-block;
+}
+
+.smiley-list-item .smiley-add {
+	padding: 0;
+}
+
+.button .dropdown-left .dropdown {
+	right: -12px;
+}
+
+.button .dropdown-right .dropdown {
+	left: -12px;
+}
+
+.button .dropdown .pointer-inner {
+	top: 1px;
+}
+
```

Comment:
![image](https://cloud.githubusercontent.com/assets/12277453/25318163/ba032d44-28a6-11e7-8393-7285d66d4a49.png)
But itz working fine for upside dropdown.
Moreover checkedtheresult changing span to div. But no use.